docs/keda-installation.md

```diff
@@ -0,0 +1,316 @@
+
+Use the `--template` flag to deploy the Keda module manifest from the beggining, or apply it using kubectl later.
+
+   ```bash
+   kyma alpha deploy --template=./template.yaml
+   ```
+Kyma installation is ready, but the module is not yet activated.
+
+   ```bash
+   kubectl get kymas.operator.kyma-project.io -A
+   ```
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME           STATE   AGE
+   kcp-system   default-kyma   Ready   71s
+   ```
+
+Keda module is a known module, but not yet activated.
+
+   ```bash
+   kubectl get moduletemplates.operator.kyma-project.io -A 
+   ```
+
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME                  AGE
+   kcp-system   moduletemplate-keda   2m24s
+   ```
+
+8.  Give Module Manager permission to install CustomResourceDefinition (CRD) cluster-wide.
+
+> **NOTE:** Module Manager must be able to apply CRDs to install modules. In the remote mode (with control-plane managing remote clusters) it gets an administrative kubeconfig, targeting the remote cluster to do so. But in the local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
+
+Run the following to make sure Module Manager's Service Account gets an administrative role:
+
+   ```bash
+   kubectl edit clusterrole module-manager-manager-role
+   ```
+
+Add the following element under `rules`:
+
+   ```yaml
+   - apiGroups:
+     - "*"
+     resources:
+     - "*"                  
+     verbs:                  
+     - "*"
+  ```
+
+> **NOTE:** This is a temporary workaround and is only required in the single-cluster mode.
+
+9. Enable Keda in the Kyma CR.
+
+   ```bash
+   kyma alpha enable module keda -c fast
+   ```
+   
+### Local mode in k3d
+
+When using a local k3d cluster, you can also use the local OCI image registry that comes with it.
+Thanks to that, you don't need to push the Keda module images to a remote registry and test the changes in the Kyma installation set up entirely on your machine.
+
+1. Clone the project.
+
+   ```bash
+   git clone https://github.com/kyma-project/keda-manager.git && cd keda-manager/
+   ```
+2. Build the manager locally and run it on the k3d cluster.
+
+   ```bash
+   make -C hack/local run-without-lifecycle-manager
+   ```
+> **NOTE:** To clean up the k3d cluster, use the `make -C hack/local stop` make target.
+
+### Install on remote Kyma runtime
+
+> **Prerequisite:** `lifecycle-manager` must be installed on the cluster (locally), or the cluster itself must be managed remotely by the central control-plane.
+
+In this section, you will learn how to install a PR version of the Keda module by a `lifecycle-manager` on a remote cluster.
+Such a setup requires OCI images for the Keda module version to be built and pushed into a public registry. Also, a module template matching the version must be available so that it can be applied on the remote cluster.
```

Comment:
Are both these sentences prerequisites to the instruction? Then add it to line 170ff. Or is this a description of what's going to happen in the subsequent instructions? Then maybe use more active voice and less passive/impersonal voice to make it clearer that this is what the user must do.